python/llm/scripts/README.md

```diff
@@ -0,0 +1,25 @@
+# README
+
+## Linux
+### CPU
+Use `python linux-env-check.py` to print the environment-related information. 
+```bash
+python linux-env-check.py
+```
+
+### XPU
```

Comment:
- use a single script on different OS, detect OS and choose different paths
- Print if hardware (e.g. XPU) is available, and if available, the detailed information. Don't use options '-xpu'

Reply:
add some examples outputs of the script.

Reply:
add try catch to ensure when error happens, it will print the error and proceed to print following sections.